README.md

```diff
@@ -12,15 +12,21 @@ To get started, go to the [quickstart](./QUICKSTART.md), or look in the [test di
 
 Distributed under the Apache 2.0 License.
 
+# Requirements
+
+To build this library locally, the following requirements necessary are :
+
+1. Daml Connect (2.0)
+2. `make` (3.x)
+3. `yq` (4.x)
```

Comment:
Is it really worth creating and maintaining these dependencies?

What do we gain by doing this, feature-wise? I'd say:
1. We are trading-off the requirement to manually download `daml-ctl`  vs having to install two new dependencies: `make` and `yq`
2. We now type `make build` instead of `daml build`.

In this light, I'm not sure whether this is a good idea.

Don't get me wrong - I see the value in this somewhere like `daml-finance` where we have multiple targets and dependencies. But feels like a stretch here.

Reply:
> Is it really worth creating and maintaining these dependencies?

We already have a dependency on `make` - notice that I'm updating that file, not creating a new one. I'm just updating the `readme` to reflect a dependency that we already have.   

> We are trading-off the requirement to manually download `daml-ctl` vs having to install two new dependencies: `make` and `yq`

Only one new dependency in `yq` as `make` was already a dependency on this project. 

> We now type `make build` instead of `daml build`.

Only if you wish to use it - nothing is stopping a developer from manually downloading the library and running `daml build`.

The point of this change is not just to automatically download changes in dependencies as part of CI, but to be able to **run the same build locally as remotely**. As you can see in this pull request, `CircleCI`s config has been updated to  execute `make build` and `make test` - this means a developer can locally check for issues before pushing remotely, avoiding having to debug remote builds.

Happy to discuss.

Reply:
Sorry there wasn't a chance to discuss this in the standup.

I'm not particularly against having this makefile, I take your point about being to reproduce remote builds - it's useful.

I just don't want to convolute the build process for a casual user that decides to try out the library and then thinks: "I need to install make and yq (and awk, sed, and gnu tools?)". Because none of those are actual build time dependencies.

I think my main gripe here is with the README. How would you feel about leaving it as-is? Then anyone who has `make` will know how it works and how to use it. For a casual (read: inexperienced) user, they won't care about this.

Reply:
We can discuss further during tomorrows standup (this work obviously isn't that critical). 

> I think my main gripe here is with the README. How would you feel about leaving it as-is? 

I'm fine to remove my commit on the `README`.